src/app/components/ButtonCustomization/index.tsx

```diff
@@ -11,14 +12,19 @@ interface Props {
   onClick?: () => void;
   className?: string;
   isLoading?: boolean;
+  page?: string;
+  prefix?: string;
+  id?: string;
 }
 
 export const ButtonCustomization = (props: Props) => {
-  const { children, onClick, className, isLoading } = props;
+  const { children, onClick, className, isLoading, page, prefix, id } = props;
```

Comment:
What is this? Why Link wrap on the button, these 2 have different components. Not allowed to share, violate the rules **Single responsibility principle**, where the button wraps a button, not a link.

Reply:
well,from now on i know

Reply:
![339827453_208882511784271_3529495507787710818_n](https://user-images.githubusercontent.com/102164071/230054085-ef2918a5-4c42-40b6-867d-08fde35f439d.png)

Reply:
I learned about SRP and now I have fixed it like this, is that correct?

Reply:
okay cool